src/components/App.js

```diff
@@ -246,24 +254,39 @@ class App extends Component {
         />
 
         {/* Top-bar components */}
+        <Playlist
+          data={this.state.data}
+          toggleDisplayPlaylist={this.toggleDisplayPlaylist}
+        />
         <div className="top-bar">
           <Player
             percentComplete={this.state.percentComplete}
             handleTrackScrub={this.handleTrackScrub}
             percentBuffered={this.state.percentBuffered}
           />
-          <button
-            onClick={this.toggleFavourite}
-            title="Toggle Favourite"
-            className="btn"
-          >
-            <FaStar size={iconSize} className="icons" />
-          </button>
-          <MenuItemInfo />
-          <MenuItemPreferences
-            togglePreferenceState={this.togglePreference}
-            preferenceState={this.state.preference}
-          />
+          <div className="top-left">
+            <button
+              onClick={this.toggleDisplayPlaylist}
+              title="Playlist"
+              className="btn"
```

Comment:
The [default type of button is `submit`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type) which isn't usually what you want unless the button was inside a form. In this case `<button type="button">` is the most correct. This will apply to practically all buttons on the page.

Reply:
So I just need to add something like 
```js
<button
  type="button"
  ...
>
```

Reply:
Yep! Just that easy.

Reply:
Amazing, thanks for letting me know this! :sparkles: